projects/io-backend-projects/provider.tf

```diff
@@ -0,0 +1,21 @@
+terraform {
+  backend "azurerm" {
+    resource_group_name  = "io-infra-rg"
+    storage_account_name = "ioinfrastterraform"
+    container_name       = "azuredevopsstate"
+    key                  = "io-backend-projects.terraform.tfstate"
```

Comment:
Should we have a common module to import?

Reply:
Terraform state right dimension it's our choice.
`backend` remote state does't support variables or functions, so we need to manual `key` configuration.
I supposed to have a few projects splitted by "area", for example frontend, backend, infrastructure, packages, ecc.
Each github repo has his folder inside the project with the pipelines.

Anyway take this repo as a PoC and to try add a step by step configuration.